scorecard.js

```diff
@@ -0,0 +1,67 @@
+const Frame = require("./frame");
+// const readline = require("readline");
+// const rl = readline.createInterface({
+//   input: process.stdin,
+//   output: process.stdout,
+// });
+
+class Scorecard {
+  constructor() {
+    this.frames = [];
+    this.bonusRolls = 0
+    this.bonus = 0
+  }
+
+  getFrames() {
```

Comment:
Good, clear naming of methods within this class - makes it easy to follow what each is doing